### PR TITLE
Removing configuration manager

### PR DESCRIPTION
**Pull request by James-Lappin**

### CODE REVIEW

File: JudoPayDotNet/Http/Client.cs [hunk 1]
```diff
@@ -6,46 +6,46 @@ namespace JudoPayDotNet.Http
 {
     internal class Client : IClient
     {
-        private readonly Connection _connection;
+        public Connection Connection { get; }
```
Comment (mikehancock) [inline JudoPayDotNet/Http/Client.cs]: does this really need to be public? if you just need it in tests it can be internal

File: JudoPayDotNet/Http/Connection.cs [hunk 2]
```diff
@@ -23,7 +23,7 @@ public class Connection
     {
         private readonly IHttpClient _httpClient;
 
-        private readonly Uri _baseAddress;
+        public readonly Uri BaseAddress;
```
Comment (mikehancock) [inline JudoPayDotNet/Http/Connection.cs]: same here, i don't see why this would need to be public

File: JudoPayDotNet/Http/Client.cs (hunk 1 above)
Comment (James-Lappin) [reply JudoPayDotNet/Http/Client.cs]: This needs to be described on the IClient interface as an IClient is used in the tests.

File: JudoPayDotNet/Http/Connection.cs (hunk 2 above)
Comment (James-Lappin) [reply JudoPayDotNet/Http/Connection.cs]: I was unsure whether someone debugging would ever want to know the URL they are processing against. However this sounds like a bit of an edge case so I have made it internal

File: JudoPayDotNet/Http/Client.cs (hunk 1 above)
Comment (mikehancock) [reply JudoPayDotNet/Http/Client.cs]: @James-Lappin we already have InternalsVisibleTo set for the tests project, so internal would work
